pyproject.toml

```diff
@@ -84,6 +84,7 @@ dev = [
   "google-cloud-aiplatform>=1.3",
   "anthropic",
   "prometheus_client",
+  "respx",
```

Comment:
I guess this goes back to one of my questions but now it feels like there's no real difference between 'dev' and 'test' if added here?

Reply:
Reverted in df382cf6. We don't have a strong distinction right now.